examples/isolated-variables/isolated_variables.description

```diff
@@ -0,0 +1,10 @@
+// An `isolated` variable's mutable state is isolated from the rest of the program. The
+// mutable state reachable from such a variable is only reachable from the variable itself,
```

Comment:
```suggestion
// mutable state is reachable from such a variable only from the variable itself
```

Reply:
I'm not sure if this change conveys the original message.

Reply:
Having the "reachable" occurrence doesn't sound correct in this sentence.

Reply:
Any other suggestions? I'm not sure how we can convey this without repeating it.

Reply:
" The mutable state is reachable from such a variable only from itself."

Reply:
The mutable state here is specifically the mutable state that is reachable from the variable. I'm not sure with this change if it is clear what mutable state we are talking about.

Reply:
Let's go ahead with how this is defined in the spec.